src/fu-util-common.c

```diff
@@ -269,18 +269,11 @@ gchar *
 fu_util_get_client_version (void)
 {
 	GString *string = g_string_new ("");
-
 	g_string_append_printf (string,
 				"%i.%i.%i",
 				FWUPD_MAJOR_VERSION,
 				FWUPD_MINOR_VERSION,
 				FWUPD_MICRO_VERSION);
-#ifdef FWUPD_DIRTY_VERSION
```

Comment:
If you prefer to go this approach with release versions, I do think we still need to show the commit version in the --version.  It's very useful to know if someone is testing with a patch or without a patch.

Reply:
Can we wire in a git check into meson easier than using an external helper? I have to switch to daddy mode now, but can look again on Monday.

Reply:
Fixed.